sequel/benchmarks/bm_sequel_destroy.rb

```diff
@@ -0,0 +1,30 @@
+require 'bundler/setup'
+require 'sequel'
+require_relative 'support/benchmark_sequel'
+
+DB = Sequel.connect(ENV.fetch('DATABASE_URL'))
+
+DB.create_table(:users) do
+  primary_key :id
+  String :name, size: 255
+  String :email, size: 255
+  DateTime :created_at, null: true
+  DateTime :updated_at, null: true
+end
+
+class User < Sequel::Model
+  self.raise_on_save_failure = true
+  self.unrestrict_primary_key
```

Comment:
For our case id remains the primary key, so we may as well skip the setters for primary key.